proposals/vector_to_regular_points.json

```diff
@@ -23,12 +23,22 @@
         },
         {
             "name": "distance",
-            "description": "Defines the minimum distance in the unit of the reference system that is required between two samples generated *inside* a single geometry.\n\n- For **polygons**, the distance defines the cell sizes of a regular grid that starts at the upper-left bound of each polygon. The centroid of each cell is then a sample point. If the centroid is not enclosed in the polygon, no point is sampled. If no point can be sampled for the geometry at all, the first coordinate of the geometry is returned as point.\n- For **lines** (line strings), the sampling starts with a point at the first coordinate of the line and then walks along the line and samples a new point each time the distance to the previous point has been reached again.\n- For **points**, the point is returned as given.",
+            "description": "Defines the minimum distance in meters that is required between two samples generated *inside* a single geometry. If the unit of the spatial reference system is not meters, a `UnitMismatch` error is thrown. Use ``vector_reproject()`` to convert to a suitable spatial reference system.\n\n- For **polygons**, the distance defines the cell sizes of a regular grid that starts at the upper-left bound of each polygon. The centroid of each cell is then a sample point. If the centroid is not enclosed in the polygon, no point is sampled. If no point can be sampled for the geometry at all, the first coordinate of the geometry is returned as point.\n- For **lines** (line strings), the sampling starts with a point at the first coordinate of the line and then walks along the line and samples a new point each time the distance to the previous point has been reached again.\n- For **points**, the point is returned as given.",
             "schema": {
                 "type": "number",
                 "minimumExclusive": 0
             }
         },
+        {
+            "name": "unit",
+            "description": "Specifies the unit the distance is given in. The only allowed value is meters (`m`), which is the default value. This parameter exists to be future-proof and might be extended in the future.",
+            "optional": true,
+            "default": "m",
```

Comment:
Only a nitpick, but I'd prefer this to be `meters` to be extra unambiguous.

Reply:
We used `m` elsewhere and it's the official SI unit so we should stick to it.

Reply:
Ah good point!